azure-pipelines.yml

```diff
@@ -0,0 +1,59 @@
+############################################################################################
+#  This Azure Pipelines was created after much time expend exploring the documentation
+#  available online.
+#
+#  Use it at your own risk.
+#
+#  Here is how it works:
+#  - We configure it to run for any branch but only for PRs to master.
+#  - The OFFICIAL_REPO variable avoids issues with Forks.
+#  - The DeployControl job is responsible for defining a variable that is used
+#  by the other deployment jobs to ensure that we only deploy for the OFFICIAL_REPO
+#  and not for PRs.
+#  - We currently use the template to run the tests (See azure-test-template.yml).
+#  - After the tests pass 3 Publish jobs go into action:
+#      - Documentation
+#        This job is executed on commits to master and tags.
+#        It deployes the new docs to the gh-pages branch at GitHub.
+#      - PyPI
+#        This job is executed on tags only.
+#        It deploy the package to the Python Package Index.
+#      - Anaconda
+#        This job is executed on commits to master and tags.
+#        It deploy the package to the DEV or TAG channels at Anaconda Cloud.
+############################################################################################
+
+trigger:
+  branches:
+    include:
+    - '*' # Build for all branches if they have a azure-pipelines.yml file.
+  tags:
+    include:
+    - 'v*' # Ensure that we are building for tags starting with 'v' (Official Versions)
+
+# Build only for PRs for master branch
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - master
+      - develop
+
+variables:
+  OFFICIAL_REPO: 'j9ac9k/pyqtgraph'
```

Comment:
I guess this should change before merge?

Reply:
yup, update made (also cleaned up the comment code top of the non-templates file)